docs/ref/django-admin.txt

```diff
@@ -1543,6 +1543,7 @@ Use the ``--locale`` or ``-l`` option to specify the locale to process.
 If not provided all locales are processed.
 
 .. django-admin-option:: --noinput
+.. django-admin-option:: --no-input
```

Comment:
Actually, maybe it's better to document the new alias using a sentence at the end of this section instead of "django-admin-option". Otherwise, this opens up the possibility to link either option which will result in inconsistent docs.